sync/patches/kube-linter/patch.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+repo_dir=$(git rev-parse --show-toplevel) ; readonly repo_dir
+script_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd ) ; readonly script_dir
+
+cd "${repo_dir}"
+
+readonly script_dir_rel=".${script_dir#"${repo_dir}"}"
+
+set -x
+cp ./vendor/kong/.kube-linter.yaml ./helm/kong-app/.kube-linter.yaml
```

Comment:
Isn't it better to do copying with vendir?

Reply:
will try! Should be cleaner if possible

Reply:
Mh was not successful doing this with vendir. It seems its not possible to have overlapping directories..